index.html

```diff
@@ -346,7 +347,7 @@ <h4 class="panel-title">
         </div>
 
         <div class="title"  ng-click="showModal('topicModal')">
-          <span class="desktop" ng-repeat="part in activeBuffer().title" ng-class="::part.classes" ng-bind-html="::(part.text | linky:'_blank':{rel:'noopener noreferrer'} | DOMfilter:'irclinky')"></span>
+          <span class="desktop" ng-repeat="part in activeBuffer().title" ng-class="::part.classes" ng-bind-html="::(part.text | DOMfilter:'irclinky')"></span>
```

Comment:
why did you remove linkification from the topic?

Reply:
also, you don't need `DOMfilter` if you're passing plain text into it

Reply:
Because I removed the old linky filter. I could have put conditionalLinky back in but upon testing I saw that links in topic worked..

Reply:
I thought irclinky turns `#asdf` into channel links..

Reply:
yeah that's what `irclinky` does, but you don't need to wrap it in `DOMfilter` if you're passing `part.text` into it, `DOMfilter` is only needed if you pass the output of another filter that creates DOM elements

Reply:
Links in the topic "work" in pull.g-b because the updater is broken by the ES6 syntax in another recent pull request, it's still using angular's `linky` :see_no_evil:

Reply:
Readded it, you are right, my testing method was wrong.